bellows/zigbee/endpoint.py

```diff
@@ -59,22 +61,29 @@ def initialize(self):
 
         self.output_clusters = sd.output_clusters
 
+        for cluster in sd.output_clusters:
+            self.add_cluster(cluster)
+
```

Comment:
Yes you are right. I just forgot to delete this before I committed. It was a test to handle the incoming ota frames. 
I will remove it.